messagix/mercury.go

```diff
@@ -53,6 +54,9 @@ func (c *Client) SendMercuryUploadRequest(ctx context.Context, threadID int64, m
 
 	_, respBody, err := c.MakeRequest(url, "POST", h, payload, types.NONE)
 	if err != nil {
+		if strings.Contains(err.Error(), "access token is no longer valid") {
+			return nil, ErrTokenInvalidated
+		}
```

Comment:
This is just checking if it's the error and returning the same error 🤔

Reply:
```suggestion
```

Reply:
🤦